website/redirects.js

```diff
@@ -0,0 +1,1234 @@
+    destination:
+      'https://learn.hashicorp.com/collections/nomad/job-failure-handling',
+    permanent: true,
+  },
+  {
+    source:
+      '/guides/operating-a-job/failure-handling-strategies/reschedule(.html)?',
+    destination:
+      'https://learn.hashicorp.com/tutorials/nomad/failures-reschedule',
+    permanent: true,
+  },
+  {
+    source:
+      '/guides/operating-a-job/failure-handling-strategies/restart(.html)?',
+    destination: 'https://learn.hashicorp.com/tutorials/nomad/failures-restart',
+    permanent: true,
+  },
+
+  {
+    source: '/guides/operating-a-job/update-strategies',
+    destination:
+      'https://learn.hashicorp.com/tutorials/nomad/job-update-strategies',
+    permanent: true,
+  },
+  {
+    source:
+      '/guides/operating-a-job/update-strategies/blue-green-and-canary-deployments(.html)?',
+    destination:
+      'https://learn.hashicorp.com/tutorials/nomad/job-blue-green-and-canary-deployments',
+    permanent: true,
+  },
+  {
+    source:
+      '/guides/operating-a-job/update-strategies/handling-signals(.html)?',
+    destination:
+      'https://learn.hashicorp.com/tutorials/nomad/job-update-handle-signals',
+    permanent: true,
+  },
+  {
+    source: '/guides/operating-a-job/update-strategies/index.html',
+    destination:
+      'https://learn.hashicorp.com/tutorials/nomad/job-update-strategies',
+    permanent: true,
+  },
+  {
+    source:
+      '/guides/operating-a-job/update-strategies/rolling-upgrades(.html)?',
+    destination:
+      'https://learn.hashicorp.com/tutorials/nomad/job-rolling-update',
+    permanent: true,
+  },
+
+  {
+    source: '/guides/operations',
+    destination:
+      'https://learn.hashicorp.com/collections/nomad/manage-clusters',
+    permanent: true,
+  },
+  {
+    source: '/guides/operations/autopilot(.html)?',
+    destination: 'https://learn.hashicorp.com/tutorials/nomad/autopilot',
+    permanent: true,
+  },
+  {
+    source: '/guides/operations/cluster/automatic(.html)?',
+    destination: 'https://learn.hashicorp.com/tutorials/nomad/clustering',
+    permanent: true,
+  },
+  {
+    source: '/guides/operations/cluster/bootstrapping(.html)?',
+    destination: 'https://learn.hashicorp.com/tutorials/nomad/clustering',
+    permanent: true,
+  },
+  {
+    source: '/guides/operations/cluster/cloud_auto_join(.html)?',
+    destination: 'https://learn.hashicorp.com/tutorials/nomad/clustering',
+    permanent: true,
+  },
+  {
+    source: '/guides/operations/cluster/manual(.html)?',
+    destination: 'https://learn.hashicorp.com/tutorials/nomad/clustering',
+    permanent: true,
+  },
+  {
+    source: '/guides/operations/federation(.html)?',
+    destination: 'https://learn.hashicorp.com/tutorials/nomad/federation',
+    permanent: true,
+  },
+  {
+    source: '/guides/operations/(index.html)?',
+    destination:
+      'https://learn.hashicorp.com/collections/nomad/manage-clusters',
+    permanent: true,
+  },
```

Comment:
This is a duplicate of https://github.com/hashicorp/nomad/pull/9036/files#diff-4704a9099921b8dd3385e629035ccd89R335